### PR TITLE
feat: text extraction sample with large language models

### PR DESCRIPTION
**Pull request by gericdong**
## Description

Fixes #b/281562687

Added text extraction sample with large language models

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by yil532 — APPROVED** [review]
LGTM

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+      String location = "us-central1";
```
Comment (averikitsch) [inline aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:100]: Should this be a variable set in the main and used on line 93 as well?

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      List<Value> instances = new ArrayList<>();
+      instances.add(instanceValue.build());
+
+      Value.Builder parameterValueBuilder = Value.newBuilder();
+      JsonFormat.parser().merge(parameters, parameterValueBuilder);
+      Value parameterValue = parameterValueBuilder.build();
+
+      PredictResponse predictResponse =
+          predictionServiceClient.predict(endpointName, instances, parameterValue);
+      System.out.println("Predict Response");
```
Comment (averikitsch) [inline aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:115]: Should you be printing something from the predictResponse object?

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+      String location = "us-central1";
+      final EndpointName endpointName =
+          EndpointName.ofProjectLocationPublisherModelName(project, location, publisher, model);
+
+      Value.Builder instanceValue = Value.newBuilder();
```
Comment (averikitsch) [inline aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:104]: A comment on what you are doing here and the second block (parameterValueBuilder) would be helpful

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+// Extractive Question Answering with a Large Language Model
+public class PredictTextExtractionSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String instance =
```
Comment (averikitsch) [inline aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:36]: Is there a link to a reference doc we can add for this object?

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java (hunk 1 above)
Comment (gericdong) [reply aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java]: It makes sense. Updated.

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java (hunk 2 above)
Comment (gericdong) [reply aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:125]: I updated the code to print out the response for now.
Currently the client library is not ready to publish a proto to process the response in a structured way.

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java (hunk 3 above)
Comment (gericdong) [reply aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:112]: Agreed. Added comments to this and the second block.

File: aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java (hunk 4 above)
Comment (gericdong) [reply aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java:38]: Added a link to the public doc about the extraction prompt design.

**Review by averikitsch — APPROVED** [review]

**Pull request merged**